src/gui/InitConfigurator.cc

```diff
@@ -1,13 +1,17 @@
 
 #include "gui/InitConfigurator.h"
+#include <qchar.h>
+#include <qlistwidget.h>
```

Comment:
These should be CamelCased and lose the .h

Reply:
Ah, missed them. VSCode insists on adding those automatically. I need to find out if there's a way to disable that (or point to the public ones).

Reply:
```suggestion
#include <QListWidget>
```